spinta/backends/postgresql/commands/init.py

```diff
@@ -84,7 +84,8 @@ def prepare(context: Context, backend: PostgreSQL, dtype: DataType):
         )
 
     column_type = types[dtype.name]
-    return sa.Column(name, column_type, unique=dtype.unique)
+    nullable = not dtype.required
+    return sa.Column(name, column_type, unique=dtype.unique, nullable=nullable)
```

Comment:
You need to add this to all other types tool. Because, currently unique constraint won't be added to types like ref, geometry and others.

Unfortunately we can't handle this at the property level, because, different types, can have different specifics, so we need to repeat this for each dtype separately.